### ISSUE TITLE
Prices by use in game 

### ISSUE THREAD

**Issue by elfideorubio** [Facepunch/RustWorkshop#108]
I have seen many list in here but they all ignore the fact that certain articles are going to sell more because of the use in game, AKA The meta, while this might change in the future we got to set fair prices for all items so they can sell almost the same amount.

I made this list because I feel like nobody wants to spend more than 5 euros on a skin, the three weeks peoriod I think is fair for everyone: Release > Increase > Increase+ Leave the store.

Note: all the prices are set in € and base on a 3 weeks period

This is a little variation that can help with items of different classes 

- Weapons

    Tier 1: Semi-Automatic Pistol, Mp5, AK47, Thompson, Bolt Rifle, 
    custom smg, salvaged pick, salvaged hatched, rocket launcher,
    metal hatched and metal pickaxe.                                                                                      3/3.6/4.5

    Tier 2 : crossbow, waterpipe and revolver, pump shotgun, f1 grenade,                             2/2.4/3
    semi-automatic rifle.

    Tier 3 : all melee weapons + misc                                                                                    0.95/1.2/1.6

- Clothing

    Tier 1 : metalmask, metalplate,hoodie, pants and bottom signs                                            2.8/3.2/4
    Tier 2 : Coffee Can Helmet, work boots, road-sign vest, collared shirt, shorts                      1.8/2/2.6
    Tier 3 : the rest                                                                                                                     0.95/1.4/2

- Deployables

    Tier 1 : Armored door, Large Wood Box, sleeping bag                                                               1.6/2/2.6
    Tier 2 : Reactive Target, Sheet Metal Door, Concrete Barricade                                           1.2/1.4/2
    Tier 3 : the rest                        0.95/1.2/1.6

**Comment by hersic** [comment]
I agree, right now people only buy skins until the first price increase and then just buy the new ones.
